app/views/govuk_publishing_components/components/docs/layout_header.yml

```diff
@@ -7,6 +7,8 @@ govuk_frontend_components:
   - header
 accessibility_excluded_rules:
   - landmark-banner-is-top-level # The header element can not be top level in the examples
+  - duplicate-id-aria # IDs will be duplicated in component examples list
```

Comment:
This works, but it looks like we're getting an error about having multiple instances of `role=banner` on one page.

```
Problem: Document must not have more than one banner landmark (landmark-no-duplicate-banner)
```

I wonder if we can do something similar to ignore these errors?